### ISSUE TITLE
Inactive title caption button hover animation broken.

### ISSUE THREAD

**Issue by AsumiLuna** [Dulappy/aero-window-manager#9]
I don't know how to properly describe this but the inactive title caption button's animations for hovering is broken, and most of the time, just flashes the button when hovered. Some animation frames seem to be skipped as well. 

https://github.com/Dulappy/aero-window-manager/assets/162660668/f1be42fc-f363-482d-bf1b-4aa98b49097b

**Comment by Dulappy** [comment]
Does this issue occur when the caption button glow is disabled?

**Comment by AsumiLuna** [comment]
Yes, this issue persists after disabling any dwm modification. disabling dwm blur glass, disabling glow, disabling thick frames and custom themes, it still flashes. 

Dwm.exe seems to also spike up to 130-170mb of ram usage on average and peaks at 200mb after injector runs, not sure if this is related, or already known.

**Comment by Dulappy** [comment]
I'll look into the original issue then. As for the memory usage, I've known about it for months and will monitor the overall system memory usage to see if this just adds to the memory reported by Task Manager (since DWM also uses a bunch of memory not displayed by Task Manager) or if AWM uses an extra 100-180MB of RAM somehow.

**Comment by RU-B0** [comment]
Is it possible that this would have something to do with Inactive Window Titlebar Button Opacity being set to 256?
